packages/bvaughn-architecture-demo/playwright/Earthfile

```diff
@@ -0,0 +1,19 @@
+VERSION 0.6
+
+install:
+    FROM  mcr.microsoft.com/playwright:v1.22.0-focal
+    RUN yarn set version 3.2.1
+    COPY . /playwright
+    WORKDIR /playwright
+    RUN yarn install
+    RUN yarn playwright:install
+
+playwright-ci:
+    FROM +install
+    RUN yarn test
+
+playwright-local:
+    FROM +install
+    RUN yarn test --update-snapshots
+    # TODO Change this structure
+    SAVE ARTIFACT ./snapshots AS LOCAL ./snapshots
```

Comment:
very cool

Reply:
Haha, it _would be_ if I could get GitHub to run `earthly` ! 😆  Thanks though

Reply:
Hm. This seems to exit too early.